### ISSUE TITLE
fix Terminess (Terminus) glyph issues

### ISSUE THREAD

**Issue by ryanoasis** [ryanoasis/nerd-fonts#23]
splitting Investigate and fix Terminess (Terminus) and ProggyClean glyph issues #16 into 2 issues (#16 and this one)

**Comment by TheRizzen** [comment]
Still no terminus patched font ? :(

**Comment by ryanoasis** [comment]
Not yet sorry 😞 

Related see #86

**Comment by kierun** [comment]
Could you add an error message to the install script? Currently, running it gives:
```
(master|✓); git log | head -1
commit c232226d395b6c067a4c9ccd65a6193ac3d7ffef
(master|✓); ./install.sh Terminus
Terminus font installed to /home/usr/.local/share/fonts
(master|✓); echo $?
0
(master|✓); ls /home/usr/.local/share/fonts
total 0
```
Which is kinda confusing…

**Comment by kierun** [comment]
Not to pester you but is there any chance of a fix soon?

**Comment by ryanoasis** [comment]
@TheRizzen Terminus (Terminess) is going to be included in 0.9.0 (the TTF version), it seems to work fine but the only problem is if you do not have antialiasing enabled: #86 

--

@kierun Really sorry about the delayed reply. Can you please try the install script again? You should have more luck with the much [improved install script](https://github.com/ryanoasis/nerd-fonts/commit/2cf5f12d1721918e7e0dd7b6fc55d23a6e473e79)

Terminus by the way isn't _currently_ being patched in master (that will be available in 0.9.0).

Here is my test of this:

```bash
ls ~/.local/share/fonts/** | grep Term | wc -l
0
./install.sh Terminus
mkdir: created directory ‘/home/ryan/.local/share/fonts/NerdFonts’
‘/home/ryan/projects/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Regular/complete/Terminess (TTF) Nerd Font Complete.ttf’ -> ‘/home/ryan/.local/share/fonts/NerdFonts/Terminess (TTF) Nerd Font Complete.ttf’
‘/home/ryan/projects/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Italic/complete/Terminess (TTF) Italic Nerd Font Complete.ttf’ -> ‘/home/ryan/.local/share/fonts/NerdFonts/Terminess (TTF) Italic Nerd Font Complete.ttf’
‘/home/ryan/projects/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/BoldItalic/complete/Terminess (TTF) Bold Italic Nerd Font Complete.ttf’ -> ‘/home/ryan/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Italic Nerd Font Complete.ttf’
‘/home/ryan/projects/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Bold/complete/Terminess (TTF) Bold Nerd Font Complete.ttf’ -> ‘/home/ryan/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Nerd Font Complete.ttf’
/home/ryan/.local/share/fonts/NerdFonts: caching, new cache contents: 4 fonts, 0 dirs
/home/ryan/.local/share/fonts/NerdFonts: caching, new cache contents: 4 fonts, 0 dirs
/var/cache/fontconfig: not cleaning unwritable cache directory
/home/ryan/.cache/fontconfig: cleaning cache directory
/home/ryan/.fontconfig: not cleaning non-existent cache directory
fc-cache: succeeded
ls ~/.local/share/fonts/** | grep Term | wc -l
4
```

**Comment by kierun** [comment]
@ryanoasis  Thank you for your time. In no way or shape do I mind a delay! Here is what I ran:

```
(master|✓); git log | head -3
commit 3efe6839b3c02aae9438471a0e3d59b38b61b3ec
Author: Ryan L McIntyre <ryanoasis@users.noreply.github.com>
Date:   Fri Nov 4 20:37:38 2016 -0400
(master|✓); git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
(master|✓); ls ~/.local/share/fonts/** | grep Term | wc -l
0
(master|✓); ./install.sh --version
Nerd Fonts installer -- Version 0.1
(master|✓); ./install.sh Terminus
cp: missing destination file operand after '/home/usr/.local/share/fonts/NerdFonts'
Try 'cp --help' for more information.
/home/usr/.local/share/fonts/NerdFonts: caching, new cache contents: 0 fonts, 0 dirs
/var/cache/fontconfig: not cleaning unwritable cache directory
/home/usr/.cache/fontconfig: cleaning cache directory
/home/usr/.fontconfig: not cleaning non-existent cache directory
fc-cache: succeeded
(master|✓); ls ~/.local/share/fonts/** | grep Term | wc -l
0
```

However, I think I know what the problem might be:

```
(master|✓); ls -a patched-fonts/Terminus
total 24K
4.0K ./  4.0K ../  8.0K LICENSE.txt  4.0K README.md  4.0K README.rst
```

So, I have no fonts there. Should I have downloaded them myself or should they be in the repo? [GutHub](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Terminus) suggest that they are not there in `master`.

**Comment by ryanoasis** [comment]
Yeah so the Terminus fonts I generated yesterday but I haven't commited yet. U will try to do that later today and I will make sure to ping here. Thanks

**Comment by ryanoasis** [comment]
Pushed the source fonts and the patched fonts (TTF version of Terminus) on 0.9.0 branch

@kierun Go ahead and give it a try if you want. Thanks.

**Comment by kierun** [comment]
Here is what I tried:

```
(0.9.0|✓); git log | head -3
commit f36d4241501fc14e22f523d3fd584afb58529480
Author: Ryan L McIntyre <ryanoasis@users.noreply.github.com>
Date:   Fri Nov 18 00:13:10 2016 -0500
(0.9.0|✓); ls ~/.local/share/fonts/** | grep Term | wc -l
0
(0.9.0|✓); ./install.sh --version
Nerd Fonts installer -- Version 0.1
(0.9.0|✓); ./install.sh Terminus
'/home/yann/src/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/BoldItalic/complete/Terminess (TTF) Bold Italic Nerd Font Complete.ttf' -> '/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Italic Nerd Font Complete.ttf'
'/home/yann/src/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Regular/complete/Terminess (TTF) Nerd Font Complete.ttf' -> '/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Nerd Font Complete.ttf'
'/home/yann/src/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Italic/complete/Terminess (TTF) Italic Nerd Font Complete.ttf' -> '/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Italic Nerd Font Complete.ttf'
'/home/yann/src/nerd-fonts/patched-fonts/Terminus/terminus-ttf-4.40.1/Bold/complete/Terminess (TTF) Bold Nerd Font Complete.ttf' -> '/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Nerd Font Complete.ttf'
/home/yann/.local/share/fonts/NerdFonts: caching, new cache contents: 4 fonts, 0 dirs
/var/cache/fontconfig: not cleaning unwritable cache directory
/home/yann/.cache/fontconfig: cleaning cache directory
/home/yann/.fontconfig: not cleaning non-existent cache directory
fc-cache: succeeded
(0.9.0|✓); ls ~/.local/share/fonts/** | grep Term | wc -l
4
(0.9.0|✓);  fc-list | grep Terminess | grep Nerd
12:445:/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Nerd Font Complete.ttf: TerminessTTF Nerd Font:style=Bold
13:485:/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Italic Nerd Font Complete.ttf: TerminessTTF Nerd Font:style=Italic
28:780:/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Nerd Font Complete.ttf: TerminessTTF Nerd Font:style=Medium
43:1467:/home/yann/.local/share/fonts/NerdFonts/Terminess (TTF) Bold Italic Nerd Font Complete.ttf: TerminessTTF Nerd Font:style=Bold Italic
```

Looks good to me!  ☺  Is there a screen shot of some fonts glyths I can do so as to verify that it all works as expected?

Many thanks indeed!

PS: Note to anyone reading this and getting weird errors: `**` is a `zsh` thing.
